pkg/auth/providers/googleoauth/goauth_provider.go

```diff
@@ -401,12 +401,14 @@ func (g *googleOauthProvider) getDirectoryService(ctx context.Context, userEmail
 		// using JWTConfigFromJSON method
 		config, err := google.JWTConfigFromJSON(jsonCredentials, admin.AdminDirectoryUserReadonlyScope, admin.AdminDirectoryGroupReadonlyScope)
 		if err != nil {
-			return nil, err
+			logrus.Error(err)
```

Comment:
The logged error isnt going to have any context to help people debugging the logs know where in the code it came from. Can you add a message that gives it context.

Same as below, but make sure the message is different for the two scenarios so that when we are looking at logs, we could distinguish between the two.